File: src/dlls/CSSPWebModels/src/WebHydrometricSite.cs
/*
 * Manually edited
 * 
 */
using CSSPDBModels;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace CSSPWebModels
{
    [NotMapped]
    public partial class WebHydrometricSite
    {
        #region Properties
        public List<WebBase> TVItemParentList { get; set; }
        public List<HydrometricSite> HydrometricSiteList { get; set; }
        public List<WebBase> TVItemHydrometricSiteList { get; set; }
        public List<RatingCurve> RatingCurveList { get; set; }
        public List<RatingCurveValue> RatingCurveValueList { get; set; }
        #endregion Properties

        #region Constructors
        public WebHydrometricSite()
        {
            TVItemParentList = new List<WebBase>();
            HydrometricSiteList = new List<HydrometricSite>();
            TVItemHydrometricSiteList = new List<WebBase>();
            RatingCurveList = new List<RatingCurve>();
            RatingCurveValueList = new List<RatingCurveValue>();
        }
        #endregion Constructors
    }
}

File: src/webs/CSSPWebToolsAng/client-apps/src/app/models/webhydrometricsite.ts
import { HydrometricSite } from './generated/HydrometricSite.model';
import { RatingCurve } from './generated/RatingCurve.model';
import { RatingCurveValue } from './generated/RatingCurveValue.model';
import { WebBase } from './webbase';

export class WebHydrometricSite {
    TVItemParentList: WebBase[] = [];
    HydrometricSiteList: HydrometricSite[] = [];
    TVItemHydrometricSiteList: WebBase[] = [];
    RatingCurveList: RatingCurve[] = [];
    RatingCurveValueList: RatingCurveValue[] = [];
}
